FB.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import Select
+from selenium.common.exceptions import TimeoutException
+import time
+class SignUpTest(unittest.TestCase):
+    def setUp(self):
```

Comment:
@ruttaba Please read about "PEP8" and always apply basic guidelines in your code. Like indentation, empty lines like after class declaration, between functions, etc.